backend/capellacollab/projects/routes.py

```diff
@@ -68,6 +68,40 @@ def get_projects(
     return projects
 
 
+@router.get(
+    "/common/{user_id}", response_model=list[models.Project], tags=["Projects"]
+)
+def get_common_projects(
+    user_for_common_projects: users_models.DatabaseUser = fastapi.Depends(
+        users_injectables.get_existing_user
+    ),
+    user: users_models.DatabaseUser = fastapi.Depends(
+        users_injectables.get_own_user
+    ),
+    log: logging.LoggerAdapter = fastapi.Depends(
+        core_logging.get_request_logger
+    ),
+) -> list[models.DatabaseProject]:
+    current_users_projects = [
+        association.project
+        for association in user.projects
+        if not association.project.visibility == models.Visibility.INTERNAL
+    ]
+    projects_for_common = [
+        association.project
+        for association in user_for_common_projects.projects
+        if not association.project.visibility == models.Visibility.INTERNAL
```

Comment:
Why do you exclude internal projects? When a user is explicitly added to a project, for example as project manager, it would be nice to see this on the profile page.